en/docs/develop/understand-ide/editors/flow-diagram-editor/ai.md

```
@@ -69,7 +69,11 @@ A data loader reads documents from disk into memory so the knowledge base can in
 
 ![Data loaders list with Text Data Loader](/img/develop/flow-design-elements/data-loaders-offered.png)
 
+<<<<<<< fix-broken-links
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)